client/src/components/Feed/Post.js

```diff
@@ -296,12 +296,12 @@ const Post = ({
           cancelText="Cancel"
         >
           <p>Are you sure you want to delete the post?</p>
-        </WebModal>
-        {postId && 
-        <StyledButtonWizard nav={<WizardFormNav />}></StyledButtonWizard>
-        }   
+        </WebModal>  
       </PostCard>
-    </>
+      {postId && 
+        <StyledWizard nav={<WizardFormNav />}></StyledWizard>
+      } 
+    </div>
```

Comment:
So wait, are you having this inside the post component? I thought we wanted it outside.

Reply:
outside the `border` of the post to be specific, but still within the component.